authors/apps/authentication/tests/test_login.py

```diff
@@ -6,6 +6,8 @@
 
 
 class UserLoginTest(BaseTest):
+    def ded(self):
```

Comment:
@BagzieGracious please through more light on the use of this function and the two lines added at the end of the file.

Reply:
oooh, initially i wanted to remove all the test and then write a dummy function to see if it will work. when i replaced back the original code i forgot to remove that function. But i have seen it and removed.